alica_designer_runtime/README.md

```diff
@@ -0,0 +1,12 @@
+# The ALICA Designer runtime
+
+- To run the designer use the docker-compose.yml file:
+    $ docker-compose up
+- The designer is web based & runs on the browser. After executing the above command, navigate to http://localhost:3000/ to run the designer
+- Create/Modify the plans & export them to the your application's etc folder
+- To generate the source code you need to download a couple of jar files:
+    1. PlanDesignerFX-Codegeneration-0.1.1.1328.jar from https://drive.google.com/file/d/158mziFym4RlPp2COBuIpzb0m90abWmvg/view?usp=sharing & place it under the alica_supplementary/alica_designer_runtime/codegen folder
+    2. alica-plan-designer-fx-default-plugin-0.1.1-SNAPSHOT.1.jar from https://drive.google.com/file/d/1Og7RLLKany6T2TKUYfUIaVVXZWfOtlJB/view?usp=sharing & place it under the alica_supplementary/alica_designer_runtime/codegen/plugins folder
```

Comment:
we can zip and keep the codegen folder on drive and ask users to download and unzip in this directory.

Reply:
done